coffee-module/coffee-module-mongodb/src/main/java/hu/icellmobilsoft/coffee/module/mongodb/extension/MongoDbClientFactory.java

```diff
@@ -126,7 +127,7 @@ protected synchronized void createMongoClientIfNotExist(MongoConfigHelper mongoC
      */
     private MongoClient createMongoClient(MongoConfigHelper mongoConfigHelper) throws BaseException {
         try {
-            log.info("MongoDB uri [{0}]", mongoConfigHelper.getUri());
+            log.info("MongoDB uri [{0}]", StringUtil.maskUriAuthenticationCredentials(mongoConfigHelper.getUri()));
```

Comment:
Erre létezik saját konfigurálható megoldásunk: https://github.com/i-Cell-Mobilsoft-Open-Source/coffee/blob/master/coffee-tool/src/main/java/hu/icellmobilsoft/coffee/tool/utils/string/StringHelper.java#L65
Azt nem hasznalható ide is?

Reply:
A StringHelper.maskPropertyValue kulcs-érték párokkal dolgozik, és az értéket az alapján maszkolja, hogy a kulcs illeszkedik-e pattern-ra, így ebben az uri esetben úgy lehet használni, hogy a kulcs és az érték is ugyanaz, és az egész értéket egy '*'-ra cseréli.

Ha az megfelelő, hogy ilyen uri esetén: [mongodb://username:password@localhost:12345]
e helyett: [mongodb://*:*@localhost:12345]
ez lesz az eredmény: [*],
akkor igen, használható.

Legyen így?

Reply:
Köszi hogy utánna játrál, megfelel úgy ahogy megvalósítottad

Reply:
köszönöm, nincs mit!